src/com/google/enterprise/gsafeed/Acl.java

```diff
@@ -70,24 +106,45 @@ public Acl setUrl(String value) {
   /**
    * Gets the value of the inheritanceType property.
    *
-   * @return possible object is {@link String}
+   * @return possible object is {@link InheritanceType}
    */
-  public String getInheritanceType() {
+  public InheritanceType getInheritanceType() {
     if (inheritanceType == null) {
-      return "leaf-node";
+      return null;
     } else {
-      return inheritanceType;
+      return InheritanceType.fromString(inheritanceType);
```

Comment:
This seems awkward. Can we marshal the enum values directly, rather than store them as strings. This post at least suggests that we can:
http://blog.bdoughan.com/2011/08/jaxb-and-enums.html